Validation/Configuration/python/ECALHCAL.py

```diff
@@ -30,7 +29,6 @@ def customise(process):
     # modify the content
 
     #process.output.outputCommands.append("keep *_simHcalUnsuppressedDigis_*_*")
-    six.next(six.iteritems(process.outputModules_()))[1].outputCommands.append("keep *_simHcalUnsuppressedDigis_*_*")
```

Comment:
should there be some added lines here to replace the removed line? I may be a bit confused what six was originally doing here.

Reply:
hum, thats problem, yes. I'll have a look and add to the next iteration